test/unit/mdc-textfield/foundation.test.js

```diff
@@ -105,6 +107,37 @@ test('#init adds mdc-textfield--upgraded class', (t) => {
   t.end();
 });
 
+test('#on input focuses if input event occurs without any other events', (t) => {
```

Comment:
micronit: `'on'` instead of `'#on`, usually we reserve `#` for testing public methods